src/Search/Kernel.php

```diff
@@ -202,7 +205,11 @@ public function dependencies(Application $app)
         };
 
         $app['limit'] = function (Application $app) {
-            return MemoryLimit::mb($app['config']['process_memory_limit']);
+            return new LoggingMiddleware(
+                new CompositeLimit(
+                    MemoryLimit::mb($app['config']['process_memory_limit']),
+                    SignalsLimit::sigterm(['SIGINT', 'SIGTERM', 'SIGHUP'])
```

Comment:
method name and parameters are in disagreement about which signals are captured?